tools/gyp/pylib/gyp/MSVSNew.py

```diff
@@ -172,7 +174,7 @@ class MSVSSolution(object):
   """Visual Studio solution."""
 
   def __init__(self, path, version, entries=None, variants=None,
-               websiteProperties=True):
+               websiteProperties=True, projects_excluded_from_build=[]):
```

Comment:
Eeekkk... Mutable default parameter https://docs.python-guide.org/writing/gotchas

Reply:
Interesting... I've been writing Python scripts for probably 9 years and I've never realised that this wasn't safe to do... learn something new every day I guess!

Reply:
These changes are (apparently) not needed, so I've reverted them in the next push.